internal/helm/controller/controller.go

```diff
@@ -49,7 +49,7 @@ type WatchOptions struct {
 	Namespace               string
 	GVK                     schema.GroupVersionKind
 	ManagerFactory          release.ManagerFactory
-	ReconcilePeriod         time.Duration
+	ReconcilePeriod         string
```

Comment:
Can we revert this? Eventually watches struct would be exposed in hybrid implementation. To remain backwards compatible, let's avoid making changes to it.